Code of conduct template set (#246)
thegooddocsproject/templates: code-of-conduct/guide-code-of-conduct-response-plan.md

# Code of Conduct response plan template guide

---
Thank you for downloading the Code of Conduct response plan template from The Good Docs Project!
We welcome any kind of feedback on our templates, both positive and negative.

> To suggest improvements, open an issue on our (https://github.com/thegooddocsproject/templates/issues)[Templates repository].

Reviewer: ```suggestion
To suggest improvements, open an issue on our [Templates repository](https://github.com/thegooddocsproject/templates/issues).

```

---

This is the guide that explains how to use the Good Docs Project Code of Conduct response plan template, which is part of the Code of Conduct template set.

The Code of Conduct template set includes:

- The following guides:
- A `template-code-of conduct.md` file that you can use to create your community's public-facing Code of Conduct.
- A `template-code-of-conduct-response-plan.md` file that you can use to create and explain the response policy and procedures your team will follow as you handle Code of Conduct incidents. This template is included in the same folder as the Code of Conduct template.

> - A `template-code-of-conduct-incident-record.md` file for a form that is filled out when a community moderator takes an incident report from a community members. This template is included in the same folder as the Code of Conduct template.

Reviewer: ```suggestion
- A `template-code-of-conduct-incident-record.md` file for a form that is filled out when a community moderator takes an incident report from a community member. This template is included in the same folder as the Code of Conduct template.

```
Sorry, I know I'm repeating this edit. I just didn't want it to get missed by mistake.

- A `template-code-of-conduct-remediation-record.md` file for a form that is filled out when a community moderator meets with a community member to explain the consequences of a Code of Conduct violation. This template is included in the same folder as the Code of Conduct template.
- The following guides:
  - `guide-code-of-conduct.md`
  - `guide-code-of-conduct-response-plan.md`
  - `guide-code-of-conduct-incident-record.md`
  - `guide-code-of-conduct-remediation-record`

> You might also consider using the `../our-team/template-our-team.md` template to let your community members know who they can contact to report a Code of Conduct violation. This document is not only necessary for Code of Conduct violations. It is a core document that helps you clearly communicate who belongs to your open source project or organization.

Reviewer: I made a comment on the "is not only necessary" wording elsewhere.

## Why do I need a Code of Conduct and a response plan?

See the `guide-code-of-conduct.md` for reasons to incorporate a Code of Conduct and response plan.

## Content of the Code of Conduct response plan template

The following sections provide guidance about how to fill out each section of the Code of Conduct response plan template.

:information_source: The Code of Conduct response plan template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text. If you use the boilerplate text, make sure you replace the project name placeholders with your own project name.

> ### About the "Introduction" section

Reviewer: This tripped me up on a few documents. Am I being obtuse or are we missing the introduction section? Ha-ha, I worked it out! The h3 here has the word "Introduction" in double quotes, consistent with the other headings. I was looking for an actual heading with that title. I suspect if we removed the double quotes from here, it would help. Alternatively, add an introduction heading to the other doc, but I suspect we don't want to do that.

The introduction section explains the purpose of the response plan document.

### About the "Community moderators" section

> In the *Community moderators* section, you can list the names of the community moderators or link to a document that lists the community moderators, such as a website that uses the (Our Team)[../our-team/template-our-team.md] template.

Reviewer: ```suggestion
In the *Community moderators* section, you can list the names of the community moderators or link to a document that lists the community moderators, such as a website that uses the [Our Team](../our-team/template-our-team.md) template.

```
fix link syntax

Alternatively, you could list the names of the community moderators and their preferred contact information, such as their Slack handles.
You could also link to a dedicated email address or contact form.

Remember to respect your community moderator's privacy by only including contact methods that go through official project channels, such as community forums or project-issued email accounts.
Don't include personal email addresses or contact information.
The goal is to strike a balance between making your moderators easy to contact while also respecting their privacy.

### About the "Contacting a community moderator" section

In the *Contacting a community moderator* section, provide clear instructions for the best way to contact the community moderators.
Filling out this section will require some pre-work on your part to clearly think through how community moderators should be contacted.

### About the "Community moderator values" section

In this section, articulate the values that your community moderators should strive to uphold.

> You can customize the provided boilerplate text or write your own.

Reviewer: query: Haven't we already said this? Isn't it true for all sections?

### About the "Requirements for community moderators" section

In this section, explain the eligibility requirements for your community moderators.

> Once again, this section will require some pre-work for you to think through what your community moderators will need to do to be prepared to serve in this role:

Reviewer: ```suggestion
This section will require some pre-work for you to think through what your community moderators will need to do to be prepared to serve in this role:

```
suggestion: remove reference to prior mention. I don't think it adds much, and it doesn't aid scan-ability.

- What qualifications should someone in this role have?
- What training will be required to serve in this role?

Community moderators do need training in handling Code of Conduct incidents in order to be effective in this role.

Some training options and resources you can consider using:

- **[The Code of Conduct book](https://frameshiftconsulting.com/resources/code-of-conduct-book/)** - This free book can act as a baseline guide for handling Code of Conduct incidents. Your community moderator team could possibly host a book club and go through the chapters together, discussing them as they go. The authors of this book also offer Code of Conduct training workshops.
- **[Mozilla's Community Participation Guidelines Enforcement](https://mozilla.teachable.com/p/cpg-training-contributors)** - This free course was developed by Mozilla for Mozilla community moderators and only takes a few hours to complete. However, be aware that this course is not as comprehensive as your community may need. The training is also specific to Mozilla's Code of Conduct, although some of its principles are general enough to apply to many projects.

> - **[Otter Tech Code of Conduct Enforcement Training](https://otter.technology/code-of-conduct-training/)** - Otter Tech is a diversity and inclusion consulting firm that offers regular workshops. The course provides a general framework for how to respond to Code of Conduct violations and provides many opportunities to role-play and practice acting as a community moderator. The Otter Tech training is excellent, but expensive and is only offered in U.S. friendly time zones.

Reviewer: ```suggestion
- **[Otter Tech Code of Conduct Enforcement Training](https://otter.technology/code-of-conduct-training/)** - Otter Tech is a diversity and inclusion consulting firm that offers regular workshops. The course provides a general framework for how to respond to Code of Conduct violations and provides many opportunities to role-play and practice acting as a community moderator. The Otter Tech training is excellent, but expensive and is only offered in U.S.-friendly time zones.

```
or United States-friendly or US-friendly or perhaps reword altogether.

> - If there are other resources that we are not aware of, please (open an issue)[https://github.com/thegooddocsproject/templates/issues] to let us know!

Reviewer: ```suggestion
- If there are other resources that we are not aware of, please [open an issue](https://github.com/thegooddocsproject/templates/issues) to let us know!

```
suggestion: correct link syntax

> In addition to these resources, we recommend holding regular practice sessions with your community moderators where you role-play what it would be like to handle an actual incident if one were to occur.

Reviewer: ```suggestion
In addition to these resources, we recommend holding regular practice sessions with your community moderators where you role-play handling an incident.

```
suggestion: I think the term "role-play" implies the words that I've removed.

> Try creating a hypothetical situation and seeing if you can take it through the entire process to its conclusion.

Reviewer: ```suggestion
Try creating a hypothetical situation and see if you can take it through the entire process to its conclusion.

```
suggestion

Author: Adopted.

### About the "Community moderator terms of service" section

In this section, describe your plan for rotating community moderators or limiting terms of service if needed.

### About the "Reviewing the Code of Conduct" section

> In the *Reviewing the Code of Conduct* section, indicate your timetable for reviewing the Code of Conduct and supporting policy documents.

Reviewer: query: sometimes we say "In this section" and sometimes we put the name in italics. Is there a style? We should be consistent throughout the document set. I'm in two minds about which one I prefer.

Author: And here I thought that I was avoiding monotony by changing it up from time to time. :)

Consider reviewing your Code of Conduct and your response plan on a yearly basis at least.
An annual review can also be a good time to check that each community moderator is familiar with your Code of Conduct policies and that they have been sufficiently trained in handling Code of Conduct incidents.
Ensure that someone in your community is responsible for ensuring the Code of Conduct is reviewed regularly.
If your community uses a calendar, add an event to your calendar to remind you to do the yearly review.

### About the "Key terms used in this document" section

This section explains the meaning of the terms used throughout the remainder of the document.
You may change these terms if you'd rather use different ones.
If you do change the terms, make sure you find and replace all instances of the old term in the document.

### About the "Handling incident reports" section

The *Handling incident reports* section includes several subsections.
It makes up the main body of this document and explains how you will handle incident reports.

The remediation process explained in this template was based on the Mozilla process for handling incidents.

> You can feel free to adapt the process for your community if needed.

Reviewer: ```suggestion
Feel free to adapt the process for your community if needed.

```
suggestion.

Author: Adopted.

Whatever you do, we want you to truly make this process your own.
Take time to really think through the logistics of how to make this process work for your community.

> :information_source: All of the sections in the Code of Conduct response plan template includes boilerplate text that you can customize or adapt, use as-is, or completely replace with your own text. If you use the boilerplate text, make sure you replace the project name placeholders with your own project name.

Reviewer: Just checking - we're repeating this on purpose?

### About the "Triage" section

The *Triage* section provides guidelines for assigning an initial severity and impact assessment.

- **Severity** refers to the overall seriousness of the behavior and the risk that behavior will be repeated.
- **Impact** refers to how public the incident was and the number of community members who were or who could have been impacted by the incident, especially members of marginalized communities.

You may use your own assessment categories and definitions if preferred.

### About the "Recommend" section

The *Recommend* section provides guidelines for recommending a suggested response to the Code of Conduct incident.

### About the "Respond" section

The *Respond* section explains how to meet with the accused individual to deliver the suggested response and behavior modification plan.

At the beginning of this phase, the investigating moderator should fill out the [Code of Conduct remediation record](code-of-conduct-remediation-record.md).

### About the "Resolve" section

The *Resolve* section explains how to conclude the Code of Conduct investigation by filing the necessary paperwork for record-keeping purposes.

> It is important to file this documentation to enable the community moderators to identify and prevent potential repeated patterns of abuse in the community.

Reviewer: Here's this use of "file" again. I tripped over it a few times in this document set, but I have commented in those other places.

Author: Changed to "record and store."

### About the "Handling incident appeals" section

The *Handling incident appeals* provides guidelines for handling incident appeals.

Filling out this section will require some pre-work on your part to clearly think through which team should best handle appeals.

### About the "Preventing conflicts of interest" section

The *Preventing conflicts of interest* section explains the conditions that can be considered a conflict of interest and the steps that should be taken if a community moderator has a conflict of interest.

## Additional resources

> The authors of this template cannot stress enough the important of ensuring your community moderators are trained in the protocol and best practices for handling Code of Conduct incidents.

Reviewer: ```suggestion
The authors of this template cannot stress enough the importance of ensuring your community moderators are trained in the protocol and best practices for handling Code of Conduct incidents.

```
suggestion (typo)

The following lists some of the resources for training your moderators that were mentioned earlier in this guide:

- [The Code of Conduct book](https://frameshiftconsulting.com/resources/code-of-conduct-book/)
- [Mozilla's Community Participation Guidelines Enforcement](https://mozilla.teachable.com/p/cpg-training-contributors)
- [Otter Tech Code of Conduct Enforcement Training](https://otter.technology/code-of-conduct-training/)

> Once again, if there are other resources that we are not aware of, please (open an issue)[https://github.com/thegooddocsproject/templates/issues] to let us know!

Reviewer: ```suggestion
Once again, if there are other resources that we are not aware of, please [open an issue](https://github.com/thegooddocsproject/templates/issues) to let us know!

```
suggestion: correct link syntax

Author: Adopted.
